x-pack/plugins/security_solution/public/kubernetes/routes.tsx

```diff
@@ -6,20 +6,20 @@
  */
 
 import React from 'react';
-import { TrackApplicationView } from '@kbn/usage-collection-plugin/public';
 import { allowedExperimentalValues } from '../../common';
 import { KubernetesContainer } from './pages';
 
 import type { SecuritySubPluginRoutes } from '../app/types';
 import { SecurityPageName } from '../app/types';
 import { KUBERNETES_PATH } from '../../common/constants';
 import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
+import { SecurityRoutePageWrapper } from '../common/components/security_route_page_wrapper';
 
 export const KubernetesRoutes = () => (
   <PluginTemplateWrapper>
-    <TrackApplicationView viewId={SecurityPageName.kubernetes}>
```

Comment:
we don't need to track collection anymore with TrackApplicationView, or it is now being handled by the  SecurityRoutePageWrapper?

Reply:
its already included internally , had a discussion with @semd